examples/seismic/tti/wavesolver.py

```diff
@@ -34,6 +34,10 @@ def __init__(self, model, geometry, space_order=4, kernel='centered',
         self.geometry = geometry
         self.kernel = kernel
 
+        if model.fs and kernel == 'staggered':
+            raise ValueError("Bad arguments in AnisotropicWaveSolver() - " +
```

Comment:
"Free surface only supported for centered TTI kernel"

Reply:
Ok